cli/tools/repl/editor.rs

```diff
@@ -334,7 +337,14 @@ impl Highlighter for EditorHelper {
                 } else if ident == *"async" || ident == *"of" {
                   colors::cyan(&line[range]).to_string()
                 } else {
-                  line[range].to_string()
+                  let next = lexed_items.get(0).map(|item|&item.inner);
+                  if matches!(next, Some(deno_ast::TokenOrComment::Token(next_token)) if matches!(next_token, Token::LParen)) {
+                    // We're looking for something that looks like a function
+                    // We use a simple heuristic: 'ident' followed by 'LParen'
+                    colors::intense_blue(&line[range]).to_string()
```

Comment:
I like it. Is there any precedence where other people are doing this too?

Reply:
Not sure, I just used this trick before for a repl I have and it worked well

Reply:
But its just a heuristic, so it only works for the *most* part, which I think its fine personally for a repl.

Reply:
Maybe we should apply same coloring as in diagnostics coming from TSC or the linter?

Reply:
Oh, by precedence I just meant someone else using this colour here. The heuristic looks good. Anyway, not a big deal.